ouroboros-consensus/bench/mempool-bench/Main.hs

```diff
@@ -49,7 +49,7 @@ main = do
             unless success exitFailure
       where
         bsss = [defaultInMemoryBSS, defaultLMDB_BSS]
-        ns   = [10_000, 20_000]
```

Comment:
This is an extra change, since running the benchmarks for `20_000` takes too long